docs/persona_paths/shop_manager.md

```diff
@@ -0,0 +1,22 @@
+# Shop manager role
+
+When you run a digital presence of your store, these three statements seem important: 
+products must go, customers must stay, and money must flow. 
```

Comment:
Is this formulation taken from out sales/marketing material? Ideally it should be consulted with them on whether it's aligned with their messaging.

Reply:
I have changed the wording to sound less blatant, and I'll ping the right people